### DISCUSSION TITLE
I have been approved but haven't received email

### DISCUSSION THREAD

**Discussion by christadrian** [community/community#77066] [GitHub Education]
### Hi everyone,

![Screenshot_20231127-070221_Chrome](https://github.com/community/community/assets/151912342/18454f57-0a07-4973-bfb4-ad3beb1a01a2)
I applied for the student developer pack but the benefits are still not available to me.Any help to resolve this.

**Comment by kur0neko** [comment]
I would suggest to contact them through email support@github.com

**Reply by christadrian** [reply]
Thanks,i think i just had to wait, It's good now.
